BTSPocket/ViewModels/TeamsViewModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TeamsViewModel.swift
+//  BTSPocket
+//
+//  Created by bts on 27/04/21.
+//
+
+import Foundation
+
+struct TeamViewModel {
+    //Fuction that comes the data of the team in BTS
+    func getTeamMembers(_ numberOfPage: Int, _ inputSearch: String?, _ completition: @escaping( (Result<PaginationAllUsers, Error>) -> Void )) {
```

Comment:
you can use an enum with arguments instead of several parameters in method 

<img width="412" alt="Screen Shot 2021-05-13 at 10 30 25 AM" src="https://user-images.githubusercontent.com/42077660/118153531-a0af8800-b3db-11eb-8007-70a7a8345ffa.png">